rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -106,13 +106,8 @@ namespace rsx
 
 			if (addr == rsx->device_addr + 0x30)
 			{
-				if (g_cfg.video.frame_limit == frame_limit_type::_ps3 && rsx->requested_vsync)
-				{
-					// Enables PS3-compliant vblank behavior
-					rsx->flip_sema_wait_val = arg;
-					rsx->wait_for_flip_sema = (sema != arg);
-				}
-
+				// HW flip synchronization related? we overwite it anyway
+				vm::write32(addr, 1);
```

Comment:
This is obviously inverted logic here. This is a frame acquire semaphore, its supposed to insert a barrier here that states "do not display the payload until this value changes".
Now, obviously this whole thing is simulated, but this is the wrong semantic here. This write is a release operation, not an acquire.

Reply:
This is still a problem.

Reply:
Moved it to semaphore_release.